src/makeInfinityAdder.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  return function adder(n) {
+    if (n === undefined) {
+      const sumToReturn = sum;
```

Comment:
It would be better in this case if the variable were call "Number", not just n. But in general all looks well!